webui test failures on nacl

Hi,

I'm trying to update the vendored pprof in the Go repository for the upcoming go 1.10 release, and the `nacl-386` and `nacl-amd64p32` trybots are failing with the following error:

```
2017/11/01 14:21:22 httptest.Server blocked in Close after 5 seconds, waiting for connections:
  *net.TCPConn 0x13fd4158 127.0.0.1:5 in state active
  *net.TCPConn 0x13fd4160 127.0.0.1:6 in state active
  *net.TCPConn 0x13fd4168 127.0.0.1:7 in state active
  *net.TCPConn 0x13fd4170 127.0.0.1:8 in state active
  *net.TCPConn 0x13fd4178 127.0.0.1:9 in state active
  *net.TCPConn 0x13fd4180 127.0.0.1:10 in state active
  *net.TCPConn 0x121100d0 127.0.0.1:3 in state active
  *net.TCPConn 0x13fd4150 127.0.0.1:4 in state active

panic: test timed out after 3m0s

[stacktrace]
```

Full logs here: 

- https://storage.googleapis.com/go-build-log/c5ef43f3/nacl-386_eb610964.log
- https://storage.googleapis.com/go-build-log/c5ef43f3/nacl-amd64p32_290b8dcf.log

### What version of pprof are you using?

I vendored revision 2fb0552a69e2f33975f90698a2a1155e8c1a46f8.

### What operating system and processor architecture are you using?

The test is failing on the `nacl-386` and `nacl-amd64p32` trybots.

### What did you do?

I vendored pprof into the go repository and uploaded the change.

### What did you expect to see?

Trybots passing.

### What did you see instead?

nacl trybots failing.

--- a/internal/driver/webui_test.go
+++ b/internal/driver/webui_test.go
@@ -28,9 +28,14 @@ import (
 
 	"github.com/google/pprof/internal/plugin"
 	"github.com/google/pprof/profile"
+	"runtime"
 )
 
 func TestWebInterface(t *testing.T) {
+	if runtime.GOOS == "nacl" {
+		t.Skip("test assumes tcp available")
+	}
+
 	prof := makeFakeProfile()
 
 	// Custom http server creator

